locations/Bahraich/inputs/impact/finance_inputs.yaml

```diff
@@ -0,0 +1,46 @@
+################################################################################
+# finance_inputs.yaml - Finance input information.                             #
+#                                                                              #
+# Author: Phil Sandwell, Ben Winchester                                        #
+# Copyright: Phil Sandwell & Ben Winchester, 2021                              #
+# Date created: 14/07/2021                                                     #
+# License: Open source                                                         #
+################################################################################
+
+discount_rate: 0.1 # Fraction between 0 and 1.
+general_o&m: 500 # [$ p.a.]
+misc_costs: 0 # [$/kW]
+bos:
+  cost: 200 # [$/kW]
+  cost_decrease: 2 # [% p.a.]
+diesel_fuel:
+  cost: 0.9 # [$/litre]
+  cost_decrease: -1 # [% p.a.]
+diesel_generator:
+  cost: 200 # [$/kW]
+  installation_cost: 50 # [$/kW]
+  installation_cost_decrease: 0 # [% p.a.]
+  o&m: 20 # [$/kW p.a.]
+  cost_decrease: 0 # [% p.a.]
+grid:
+  cost: 0.01 # [$/kWh]
+  extension cost: 5000 # [$/km]
+  infrastructure_cost: 2000 # [$]
+households:
+  connection_cost: 100 # [$/household]
+  kerosene_cost: 0.008 # [$/hour]

```

Comment:
I can see why you've put this here, but I think there should be a new `kerosene` section with a single entry for `kerosene_cost`

Reply:
Have done :smile: This now matches the `diesel` etc.:
```
kerosene:
  cost: 0.008 # [$/hour]
```